resources/views/partials/nav-partials.blade.php

```diff
@@ -0,0 +1,20 @@
+
+<nav class="navbar navbar-default" style="margin-bottom: 0;">
+    <div class="col-xs-2" style="padding: 10px;">
+        <img alt="logo" class="img-fluid" src="{{ asset('css/img/Journal-app.png') }}" style="width: 150px;">
```

Comment:
Please remove css from the image path. `src="{{ asset(img/Journal-app.png') }}"`